File: python/bc1961.py
res = "YOU WIN"
aPu, qCa = map(int, input().split())
sCa = list(map(int, input().split()))

for c in range(0, qCa - 1):
    if abs((sCa[c] - sCa[c + 1])) > aPu:
        res = "GAME OVER"
        break

print(res)

File: cpp/bc1961.cpp
#include <iostream>
#include <sstream>

using namespace std;

int main()
{
	int aPu, qPu, i = 0;
	string res = "YOU WIN", sVa, token;
	
	cin >> aPu >> qPu;
	
	cin.ignore();
	
	int sPu[qPu*2];
	
	getline(cin, sVa);
	
	stringstream ss(sVa);
	
	while(ss >> token)
	{
	    sPu[i] = stoi(token);
	    i++;
	}
	
	for(int j = 0; j < qPu-1; j++)
	{
	    if(abs(sPu[j] - sPu[j+1]) > aPu)
	    {
	        res = "GAME OVER";
	        break;
	    }
	}
	
	cout << res << endl;
}
